PowerShell.Common.props

```diff
@@ -31,6 +31,9 @@
       <PSCoreAdditionalCommits>$([System.Text.RegularExpressions.Regex]::Match($(PowerShellVersion), $(RegexGitVersion)).Groups[2].Value)</PSCoreAdditionalCommits>
       <PSCoreCommitSHA>$([System.Text.RegularExpressions.Regex]::Match($(PowerShellVersion), $(RegexGitVersion)).Groups[3].Value)</PSCoreCommitSHA>
 
+      <!--
+            Caution! PSVersionInfo.cs is very sensetive to the format of this string.
```

Comment:
It's better to be more specific about what depends on this formatted version string here. How about
```
Caution! PSVersion and GitCommitId from PSVersionInfo.cs depend on the format of this version string.
```

Reply:
Done.